thoth/license_solver/solver.py

```diff
@@ -176,7 +176,7 @@ def get_file(self, file: str) -> None:
         if os.path.isfile(file) and file.lower().endswith(".json"):
             self._files_list.append(file)
         else:
-            _LOGGER.warning("wrong format you can insert only .json file: ", file)
+            _LOGGER.warning(f"wrong format you can insert only .json file: {file}")
```

Comment:
```suggestion
            _LOGGER.warning("wrong format you can insert only .json file: %s", file)
```

Reply:
This will be changed in another pull request for all files where it was used wrong